plugins/module_utils/kibana.py

```diff
@@ -28,6 +28,12 @@ def __init__(self, module):
     self.validate_certs = module.params.get('verify_ssl_cert')
     self.version = None # this is a hack to make it so that we can run the first request to get the clutser version without erroring out
     self.version = self.get_cluster_version()
+    self.integration_name = self.module.params.get('integration_name')
+    self.agent_policy_id = self.module.params.get('agent_policy_id')
+    self.agent_policy_name = self.module.params.get('agent_policy_name')
+    self.agent_policy_desc = self.module.params.get('agent_policy_desc')
+    self.pkg_policy_name = self.module.params.get('pkg_policy_name')
+    self.pkg_policy_desc = self.module.params.get('pkg_policy_desc')
```

Comment:
Sorry, I realize my last comment on this wasn't clear. These should go in the `init` inside the module class **unless** they are used by all modules that use this module_util.

Reply:
I am not sure I understand what you are asking for.  Please check code and provide feedback.